index.ts

```diff
@@ -300,19 +300,17 @@ commander
             q: 'mimeType="application/vnd.google-apps.script"',
           });
           const files = data.files;
-          const fileIds:string[] = [];
+          const fileIds: Array<{ name: string; value: string; }> = [];
```

Comment:
Can you not do this?
```ts
const fileIds: {
  name: string;
  value: string;
}[] = [];
```

Reply:
`tslint` gives this error: `Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.`

Reply:
https://github.com/google/clasp/blob/60393ef2f693f6f2cfa7b2b63cbbed1fa0f8438e/tslint.json#L5